src/modules/common/components/pageLoader/PageLoader.js

```diff
@@ -15,6 +15,11 @@ function PageLoader() {
           ))}
         </div>
       </figure>
+      {progress && (
```

Comment:
Probably need some check for 0 here as well? Get strange layout when 0 
![Uploading image.png…]()

Reply:
0.01 is the minimum indexing percentage I've experienced so far and it mitigates this issue else your concern is quite valid